### DISCUSSION TITLE
Actions构建app问题请教

### DISCUSSION THREAD

**Discussion by Aboceder** [tw93/Pake#726] [Q&A] [answered]
请问一下，我下载了.icns后缀的图片，想在actions构建的时候自定义图标，我该怎么设置呢，我把本地图片的地址放到[Icon, Optional]里面，构建成功但是没有文件提供我下载
![2024-06-05 at 10 02 39@2x](https://github.com/tw93/Pake/assets/53726224/ed994aa1-a322-4173-9f88-aa9de4c7055f)

**Comment by Tlntin** [comment, marked as answer]
将图片放图床，使用在线链接的方式填入表单。
